packages/widgets/src/lib/components/InferenceWidget/shared/WidgetOutputConvo/WidgetOutputConvo.svelte

```diff
@@ -26,8 +26,10 @@
 		{#each messages as message}
 			{#if message.role === "user"}
 				<WidgetOutputConvoBubble position="right" text={message.content} />
-			{:else}
+			{:else if message.role === "assistant"}
 				<WidgetOutputConvoBubble position="left" text={message.content} />
+			{:else}
+				<WidgetOutputConvoBubble position="center" text={message.content} />
 			{/if}
 		{/each}
```

Comment:
```suggestion
		{#each messages as {role, content}}
            {@const position = role === "user" ? "right" : role === "assistant" ? "left" : "center"}
            <WidgetOutputConvoBubble {position} text={content} />
		{/each}
```

I think we can use the `svelte const` to make it even shorter

Reply:
nice!

Reply:
or just `<WidgetOutputConvoBubble position={role === "user" ? "right" : role === "assistant" ? "left" : "center"} text={content} /> `